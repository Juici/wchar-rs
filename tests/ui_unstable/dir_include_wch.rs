use wchar::{include_wch, wchar_t};

const DIR: &[wchar_t] = include_wch!("../data");

fn main() {}
