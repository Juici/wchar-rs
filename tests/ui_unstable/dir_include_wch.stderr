error: couldn't read $DIR/tests/ui_unstable/../data: Is a directory (os error 21)
 --> $DIR/dir_include_wch.rs:3:38
  |
3 | const DIR: &[wchar_t] = include_wch!("../data");
  |                                      ^^^^^^^^^
